functions/Name/compute_q_matrix.c

```diff
@@ -18,13 +18,16 @@ static int cmp_mat_lex (matrix_TYP *A, matrix_TYP *B)
   int i, j;
 
   
-      for (i=0; i<A->rows; i++)
-	for (j=0; j<A->cols; j++)
-	  if (A->array.SZ[i][j] != B->array.SZ[i][j])
+      for (i=0; i<A->rows; i++) {
+	for (j=0; j<A->cols; j++) {
```

Comment:
Should we maybe indent the second `for` here as it is a nested `for` loop? Might make it easier to read the code.

Reply:
It is actually indented, but this code is using tabs, so the indentation is not properly shown here in the web UI; but it is there. Of course this is one of the things that using `clang-format` would take care of.

Reply:
I think we'll merge this now and potentially work on the code formatting in a seperate PR.